package.json

```diff
@@ -6,8 +6,9 @@
   "scripts": {
     "react-dev": "webpack --watch",
     "server": "nodemon ./server/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "seed": "psql postgres < database/schema.sql"
```

Comment:
what does the `postgres` part do? just curious

Reply:
No clue. I just read followed the link you had posted the other day.